Translation corrections (#83)
07th-mod/higurashi-console-arcs: Update/some_005.txt

//rじゃあ、本当は千紗登ちゃんは全然喜んでなくて、…だから、珠子ちゃんも冷たくて、それどころか、私のこと、…嫌なやつだって、思って……？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "じゃあ、本当は千紗登ちゃんは全然喜んでなくて、…だから、珠子ちゃんも冷たくて、それどころか、私のこと、…嫌なやつだって、思って……？",

> 			NULL, "Then Chisato-chan wasn't actually happy for me... Which would cause Tamako-chan to start acting cold, ... and hate me......?", Line_Normal);

Reviewer: ```suggestion
			NULL, "Then Chisato-chan wasn't actually happy for me... Which would cause Tamako-chan to start acting cold, ...and hate me......?", Line_Normal);
```

//rひょっとして千紗登ちゃん、…あるいは珠子ちゃんも、藤堂くんのことが好きだったのかな…？　だからこれって、二人に嫉妬されてるってことなの…？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ひょっとして千紗登ちゃん、…あるいは珠子ちゃんも、藤堂くんのことが好きだったのかな…？　だからこれって、二人に嫉妬されてるってことなの…？",

> 			NULL, "Perhaps Chisato-chan... or maybe Tamako-chan too... liked Toudou-kun...? Then did that mean they were jealous of me...?", Line_Normal);

Reviewer: ```suggestion
			NULL, "Perhaps Chisato-chan... or maybe Tamako-chan... liked Toudou-kun too...? Then did that mean they were jealous of me...?", Line_Normal);
```

//rさっきまでのぎくしゃくした雰囲気は、甘くて冷たいアイスクリームが全部、幸福の対岸へと押し流してしまった。…女の子は甘いものさえあれば生きていける。それはきっと、真理だ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "さっきまでのぎくしゃくした雰囲気は、甘くて冷たいアイスクリームが全部、幸福の対岸へと押し流してしまった。…女の子は甘いものさえあれば生きていける。それはきっと、真理だ。",

> 			NULL, "The sweet and cold ice cream completely washed away the awkward atmosphere from earlier and brought in bliss. Girls can live on sweets alone. Surely that's the truth.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The sweet and cold ice cream completely washed away the awkward atmosphere from earlier and brought me bliss. Girls can live on sweets alone. Surely that's the truth.", Line_Normal);
```

	ClearMessage();

//rはぁ～～、おいしぃ～～☆　幸福で、再び胸が満たされてゆく気分だ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "はぁ～～、おいしぃ～～☆　幸福で、再び胸が満たされてゆく気分だ。",

> 			NULL, "Oh~~, Delicious~~☆ My heart's filling with joy..", Line_Normal);

Reviewer: ```suggestion
			NULL, "Oh~~, Delicious~~☆ My heart's filling with joy.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>ＴＶアナ</color>", NULL, "<color=#f5e6d3>TV Announcer</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s04/00/vt1b_tvann1001", 256, TRUE);
	OutputLine(NULL, "『…以上のご連絡の通り、鹿骨市と周辺にお住まいの皆さんは、どうぞ冷静な対応をお願いいたします…』",

> 			NULL, "\"...As stated earlier, we ask that everyone living near of Shishibone City please remain calm...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"...As stated earlier, we ask that everyone living near Shishibone City please remain calm...\"", Line_Normal);
```

//r小さい頃、おばーちゃんの田舎に遊びに行った時に、見晴らしのいい高台の神社に行って、家族で記念写真を撮った。その写真は、今でもおばーちゃんの部屋のタンスの上に飾ってある。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "小さい頃、おばーちゃんの田舎に遊びに行った時に、見晴らしのいい高台の神社に行って、家族で記念写真を撮った。その写真は、今でもおばーちゃんの部屋のタンスの上に飾ってある。",

> 			NULL, "When I was young, I visited Grandma's hometown. I went to a shrine on scenic ground and took a commemorative photo with my family. That picture was still sitting on the dresser in Grandma's room. ", Line_Normal);

Reviewer: ```suggestion
			NULL, "When I was young, I visited Grandma's hometown. I went to a picturesque shrine on top of a hill and took a commemorative photo with my family. That picture was still sitting on the dresser in Grandma's room. ", Line_Normal);
```
